app/views/api/apidocs/pept2funct.html.erb

```diff
@@ -3,11 +3,11 @@
     <div class='page-header'>
       <h1 class='api-title'><span class="initialism">POST</span> /api/v1/pept2funct
         <br>
-        <small>Returns both the functional EC-numbers and GO-terms associated with a given tryptic peptide.</small>
+        <small>Returns the functional EC-numbers, GO-terms and InterPro entries associated with a given tryptic peptide.</small>
       </h1>
     </div>
     <p>
-      This method returns the functional EC-numbers and GO-terms associated with a given tryptic peptide. This is the same information as provided when performing a search with the <%= link_to("Tryptic Peptide Analysis", search_single_path) %> in the web interface.
+      This method returns the functional EC-numbers, GO-terms and InterPro entries associated with a given tryptic peptide. This is the same information as provided when performing a search with the <%= link_to("Tryptic Peptide Analysis", search_single_path) %> in the web interface.
```

Comment:
InterPro entries are not returned by the Unipept web interface at this moment, but we might just ignore this for now? (@bmesuere)

Reply:
+1 for ignore